### PR TITLE
RIA-7741 fixes for local deployment

### PR DESCRIPTION
**Pull request by nilay913**
Added and modified variables to allow easier local deployments.

### CODE REVIEW

File: .wa-env [hunk 1]
```diff
@@ -169,9 +170,12 @@ export TEST_LAW_FIRM_SHARE_CASE_D_PASSWORD=London03
 export TEST_JUDGE_X_USERNAME=ia.iacjudge.ccd@fake.hmcts.net
 export TEST_JUDGE_X_PASSWORD=London20
 
-export WA_TASK_DMNS_BPMNS_PATH="your path to DMNs repo"
-export WA_BPMNS_DMNS_PATH="your path to wa-standalone-task-bpmn repo"
-export IA_TASK_DMNS_BPMNS_PATH="your path to ia-task-configuration repo"
-
+export WA_TASK_DMNS_BPMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/wa-task-configuration-template"
+export WA_BPMNS_DMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/wa-standalone-task-bpmn"
+export IA_TASK_DMNS_BPMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/ia-task-configuration"
```
Comment (filippo-possenti-version1) [inline .wa-env:175]: The `mdfile` command is not normally available under Linux. Keep in mind that different suppliers use different machines and not everybody uses a Mac.

Replace it with a command that is available by default in Linux.

File: .wa-env [hunk 2]
```diff
@@ -169,9 +170,12 @@ export TEST_LAW_FIRM_SHARE_CASE_D_PASSWORD=London03
...
 
+export TEST_LAW_FIRM_ORG_SUCCESS_USERNAME=ia-law-firm-success@fake.hmcts.net
+export TEST_LAW_FIRM_ORG_SUCCESS_PASSWORD=London08
```
Comment (filippo-possenti-version1) [inline .wa-env:178]: These projects are open-source. Although I'm aware that these passwords are meaningless as they are specific to the local environment, better not putting them in code available for everybody to read.

```suggestion
export TEST_LAW_FIRM_ORG_SUCCESS_PASSWORD=<your password here>
```

File: .wa-env [hunk 3]
```diff
@@ -169,9 +170,12 @@ export TEST_LAW_FIRM_SHARE_CASE_D_PASSWORD=London03
...
+export TEST_LAW_FIRM_ORG_DELETED_USERNAME=ia-law-firm-deleted@fake.hmcts.net
+export TEST_LAW_FIRM_ORG_DELETED_PASSWORD=London09
```
Comment (filippo-possenti-version1) [inline .wa-env:180]: Same as above

```suggestion
export TEST_LAW_FIRM_ORG_DELETED_PASSWORD=<your password here>
```

File: .wa-env [hunk 4]
```diff
@@ -7,6 +7,8 @@ export OPEN_ID_IDAM_URL=http://sidam-simulator:5000
 export IDAM_USER_URL=http://localhost:5000
 export POSTGRES_PASSWORD=ccd
 export POSTGRES_USERNAME=ccd
+export POSTGRES_ADMIN_PASSWORD=postgres
```
Comment (filippo-possenti-version1) [inline .wa-env:10]: While other passwords can be removed, this one can stay here. No objection. Writing just for confirmation that this one is ok.

**Review by filippo-possenti-version1 — CHANGES_REQUESTED** [review]

File: .wa-env [hunk 5]
```diff
@@ -75,16 +77,15 @@ export IA_UPPER_TRIBUNAL_NOTICES_EMAIL=upper-tribunal-notices@example.com
 export IA_BAIL_GOV_NOTIFY_KEY=OOOOOOOOOOOOOOOO
 
 # ASB related variables
-export AZURE_SERVICE_BUS_CONNECTION_STRING=replace_me
-export AZURE_SERVICE_BUS_TOPIC_NAME=replace_me
-
-export AZURE_SERVICE_BUS_SUBSCRIPTION_NAME=replace_me
-export AZURE_SERVICE_BUS_CCD_CASE_EVENTS_SUBSCRIPTION_NAME=replace_me
+export AZURE_SERVICE_BUS_TOPIC_NAME=servicecallbacktopic
+export AZURE_SERVICE_BUS_CONNECTION_STRING="Endpoint=sb://ia-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=sharedAccessPolicy;SharedAccessKey=UipYzv1Q9Ye1Qxim8k2VA6iaHLs+sET0tXd+vMch1MI=;EntityPath=servicecallbacktopic"
```
[the remark this answers is not in the document]
Comment (nilay913) [reply .wa-env:81]: I got sent the AZURE_SERVICE_BUS_CONNECTION_STRING value by Srihari

[the remark this answers is not in the document]
Comment (nilay913) [reply .wa-env:81]: This connection string is obtained from SharedAccessPolicy -> Primary Connection String at https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/ia-case-payments-api-demo/providers/Microsoft.ServiceBus/namespaces/ia-servicebus-demo/topics/servicecallbacktopic/saskey, I will update the README to include this and remove this key from the env variables

File: .wa-env [hunk 6]
```diff
@@ -75,16 +77,15 @@ export IA_UPPER_TRIBUNAL_NOTICES_EMAIL=upper-tribunal-notices@example.com
...
+export AZURE_SERVICE_BUS_SUBSCRIPTION_NAME=thomasc_local
+export AZURE_SERVICE_BUS_CCD_CASE_EVENTS_SUBSCRIPTION_NAME=thomasc_local
```
[the remark this answers is not in the document]
Comment (nilay913) [reply .wa-env:83]: This actually isnt specific to local, in the azure servicecallbacktopic, it is the only subscription there other than default.

File: .wa-env (hunk 2 above)
Comment (nilay913) [reply .wa-env:178]: Passwords are already in this file, and these specific ones are pulled from the .local_env file

File: .wa-env (hunk 3 above)
Comment (nilay913) [reply .wa-env:180]: same as above

File: .wa-env [hunk 7]
```diff
@@ -75,16 +77,15 @@ export IA_UPPER_TRIBUNAL_NOTICES_EMAIL=upper-tribunal-notices@example.com
...
 
 export AZURE_SERVICE_BUS_FEATURE_TOGGLE=true
 export WA_DLQ_PROCESS_ENABLED=true
-export AZURE_SERVICE_BUS_MESSAGE_AUTHOR=replace_me
-export LAUNCH_DARKLY_ACCESS_TOKEN=replace_me
+export AZURE_SERVICE_BUS_MESSAGE_AUTHOR=thomas
+export LAUNCH_DARKLY_ACCESS_TOKEN=replace_me5
```
[the remark this answers is not in the document]
Comment (nilay913) [reply .wa-env:93]: as above

File: .wa-env (hunk 1 above)
Comment (nilay913) [reply .wa-env]: replaced with a check if they're in profile and if not then to add them (and info what path needs to be added)

**Pull request closed without merging**